sci-libs/vtk/vtk-9.0.1.ebuild

```
@@ -0,0 +1,546 @@
+# Copyright 1999-2021 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+# TODO:
+# - add USE flag for remote modules? Those modules can be downloaded
+#	properly before building.
+
+PYTHON_COMPAT=( python3_{7,8,9} )
+WEBAPP_OPTIONAL=yes
+WEBAPP_MANUAL_SLOT=yes
+
+# qmake-utils
+inherit check-reqs cmake cuda flag-o-matic java-pkg-opt-2 python-single-r1 toolchain-funcs virtualx webapp
+
+# Short package version
+SPV="$(ver_cut 1-2)"
```

We try to avoid single-letter deviations from standard variables in case they're used in future. Also, MY_* is canonical for P/PN/PV etc.

```suggestion
MY_PV="$(ver_cut 1-2)"
```

---

The `SPV` is from the old 8.2.0 release, not my idea. I can change it if you want.